base/apicast-operator/deploy-apicast.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -exuo pipefail
+command -v envsubst
+
+FILE_ROOT=${BASH_SOURCE%/*}
+NAMESPACE=${NAMESPACE:="apicast-ga"}
```

Comment:
since there are no special requirements on NAMESPACE (pull-secret pre-creation) this script should create it too